src/javascript/binary/pages/portfiliows.js

```diff
@@ -40,11 +40,12 @@ var PortfolioWS =  (function() {
         }
```

Comment:
note that there's a typo in the file name - should be portfoliows.js not portfiliows.js